hack/tools/go.mod

```diff
@@ -1,41 +1,38 @@
 module github.com/cert-manager/istio-csr/hack/tools
 
-go 1.20
-
-// oras v1.2 is incompatible with docker/docker v1.24 helm does not work with v2 of oras
-// remove the replace once https://github.com/helm/helm/issues/11821 is fixed
-replace github.com/docker/docker => github.com/docker/docker v23.0.1+incompatible
```

Comment:
note: the linked issue hasn't been resolved, but removing this replace statement worked fine as far as I can tell!